src/main/resources/i18n/Resources_zh_Hans.properties

```diff
@@ -1,10 +1,7 @@
 # suppress inspection "UnusedProperty" for whole file
-#TODO translate
-loginDialogLabelUsingCustomCredentials=Using own Google API secret (safety warnings expected)
-#TODO translate
-loginDialogLabelUsingDefaultCredentials=You can use own Google API secret to increase limits
-#TODO translate
-loginDialogConfigureButtonText=Configure...
+loginDialogLabelUsingCustomCredentials=使用自己的Google API Client Secret(会有安全提示)
+loginDialogLabelUsingDefaultCredentials=使用自己的Google API Secret来提高上限
+loginDialogConfigureButtonText=设置...
 googleStorageWarning=所有上传文件皆使用原始质量大小，会占用Google账号容量。
 # this is a tooltip that pops up when Google API quota is exceeded; the app then slows down by applying exponential backoff; the backoff delay in seconds
 # is appended to the phrase below
```

Comment:
Thanks, there's another TODO just below here - could you also translate that one? It's an addition to the existing phrase.